davici.c

```diff
@@ -156,7 +127,6 @@ static int connect_in_and_fcntl(int fd, const char *path)
 
 	if(connect(fd, (struct sockaddr*)&addr, sizeof(addr)) < 0)
 	{
-		fprintf(stderr, "AF_NET connection failed.\n");
```

Comment:
Any reason to remove the fprintfs with error information?

Likewise for all fprintf's that have been removed

Reply:
I had thought they shouldn't be in the production code, we should just return the errno. There are no other printf type statements in the existing code.

Reply:
I did the commits in the wrong order and thought the fprintfs were original code (i mentioned in an overall comment), so yes remove them